README.md

```diff
@@ -1 +1,28 @@
 # Stock-Densities
+
```

Comment:
Super sick idea to have OKRs in readme, def stealing this for future projects :D